demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -90,6 +93,10 @@ def run_validations(self) -> int:
                         in filtered_content_objects_for_validator
                     ]
                 try:
+                    validation_results = self.filter_validation_results(
```

Comment:
I think maybe it would be more efficient to check the the error_code is part of the constant from here, before calling filter_validation_results